T2CC_APP/andriod app/COSC612StudentAndriod/app/src/main/res/layout-land/activity_main.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/username"
```

Comment:
this view seems to be called username, we should call it something like loginScreen or homePage. Something representation of what it is. It makes things easier for the backend developer